nbextensions/appCell2/widgets/appCellWidget-fsm.js

```diff
@@ -34,24 +165,6 @@ define([], () => {
                     name: 'runApp',
                     disabled: true,
                 },
-                elements: {
-                    show: [],
-                    hide: [
-                        'internal-error',
-                        'parameters-group',
-                        'output-group',
-                        'parameters-display-group',
-                        'exec-group',
-                    ],
-                },
-                appStatus: {
-                    classes: ['kb-app-status-default'],
-                    icon: {
-                        type: 'minus',
-                    },
-                },
-                label: 'new',
-                message: '',
```

Comment:
only `ui.tabs` and `ui.actionButton` are used, so the rest of this can disappear.